docs/VrfVirtualCircuit.md

```diff
@@ -18,15 +18,16 @@ Name | Type | Description | Notes
 **Status** | Pointer to [**VrfVirtualCircuitStatus**](VrfVirtualCircuitStatus.md) |  | [optional] 
 **Subnet** | Pointer to **string** | The /30 or /31 subnet of one of the VRF IP Blocks that will be used with the VRF for the Virtual Circuit. This subnet does not have to be an existing VRF IP reservation, as we will create the VRF IP reservation on creation if it does not exist. The Metal IP and Customer IP must be IPs from this subnet. For /30 subnets, the network and broadcast IPs cannot be used as the Metal or Customer IP. | [optional] 
 **Tags** | Pointer to **[]string** |  | [optional] 
-**Vrf** | Pointer to [**Vrf**](Vrf.md) |  | [optional] 
+**Type** | Pointer to [**VrfIpReservationType**](VrfIpReservationType.md) |  | [optional] 
```

Comment:
Oh that's weird.  Unlike the Java generator, the Go generator doesn't collapse the virtual circuit enums, but it _does_ see a different model with the same contents (VrfIpReservationType is also an enum with `vrf` as the only allowed value) and use that instead of creating a dedicated, duplicate model.

The end result is functionally the same but it might be nice to be more intentional about it...maybe a single `VrfType.yaml` schema that we ref in multiple places in the schema, or a bunch of duplicate `<thing>Type.yaml` to force generators to create duplicate models.